netbout-web/src/main/java/com/netbout/rest/account/TkSaveEmail.java

```diff
@@ -78,7 +80,18 @@
      * @param bse Base
      */
     TkSaveEmail(final Base bse) {
+        this(bse, !Manifests.read("Netbout-Version")
```

Comment:
@dmzaytsev please follow http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation